authors/settings.py

```diff
@@ -43,12 +43,15 @@
     'authors.apps.authentication',
     'authors.apps.core',
     'authors.apps.profiles',
+    'authors.apps.articles',
+    'taggit',
+    'taggit_serializer',
 ]
 
 MIDDLEWARE = [
     'django.middleware.security.SecurityMiddleware',
     'django.middleware.security.SecurityMiddleware',
-    'whitenoise.middleware.WhiteNoiseMiddleware',
+    # 'whitenoise.middleware.WhiteNoiseMiddleware',
```

Comment:
Could there be any specific reason why you commented out `whitenoise.middleware.WhiteNoiseMiddleware`

Reply:
Fixed